docs/clients/README.md

```diff
@@ -0,0 +1 @@
+# Loki Clients
```

Comment:
I think here we should talk about which alternatives depending on your deployment you should choose and why.

Reply:
We should also mention that if you use the same relabeling in promtail as with prometheus you can switch easily between metrics and log in explore. Not sure this is the right place but I think we have to advocate about this.